mongodb/templates/systemd_local.conf.j2

```diff
@@ -0,0 +1,6 @@
+{%- for section in mongodb_systemd_overrides.items() %}
```

Comment:
The `.items()` call returns a 2-tuple of the key and the value for the dictionary object, so this would be something like:
`{% for section, settings in mongodb_systemd_overrides.items() %}
[{{ section }}]
{% for k, v in settings.items() %}
{{ k }}={{ v }}
{% endfor %}
{% endfor %}

Reply:
OK, just changed it.